docs/plugin_developers_guide/11_WSL2.rst

```diff
@@ -0,0 +1,328 @@
+.. Developing with Windows Subsystem for Linux (WSL2)
+
+.. |br| raw:: html
+
+   <br />
+
+.. Images
+
+.. Links in new tabs
+
```

Comment:
Add links here which we have used in the documentation.

Reply:
Here my point is inline links open in same browser tab. It always be good to open link in separate tabs so that we can switch the tabs instead of browser back button when we read the documentation.

Reply:
We will revisit across the doc pages if really required to change the behavior. You can go ahead with the given changes.